sound/soc/sof/Kconfig

```diff
@@ -128,6 +128,14 @@ config SND_SOC_SOF_DEBUG_ENABLE_DEBUGFS_CACHE
 	  Say Y if you want to enable caching the memory windows.
 	  If unsure, select "N".
 
+config SND_SOC_SOF_DISABLE_DMA_TRACE
+	bool "SOF disable DMA-based trace"
+	help
+	  This option disables the firmware trace completely. This
```

Comment:
May be not accurate with "disables the firmware trace completely", it's better that we can still get error trace from memory window or mailbox if DMA trace is disabled, and these can be other kinds of firmware trace?

Reply:
yes good comment

Reply:
@keyonjie  DMA TRACE also can be disabled by FW by disabling TRACE_CONFIG, so your idea is not always true.